test/visual.test.js

```diff
@@ -0,0 +1,82 @@
+import Promise from 'bluebird';
+import test from 'tape';
```

Comment:
Mocha was struggling with the way I was autogenerating tests in an async code (wait for browser)